duties.py

```diff
@@ -355,7 +355,7 @@ def format(ctx):  # noqa: W0622 (we don't mind shadowing the format builtin)
         title="Removing unused imports",
         pty=PTY,
     )
-    ctx.run(f"isort -y -rc {PY_SRC}", title="Ordering imports", pty=PTY)
+    ctx.run(f"isort {PY_SRC}", title="Ordering imports", pty=PTY)
```

Comment:
I guess version 5 now reorders imports in python files by default, without the need for these options?

Reply:
Yes https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0/#migrating-cli-options

Reply:
Yup, got my answer in the changelog: https://github.com/pycqa/isort/blob/master/CHANGELOG.md#500-penny---july-4-2020